java10/session-1-Local-Variable-Type-Inference/04_Full_uses/README.md

```diff
@@ -1,35 +1,33 @@
 **Local Variable Type Inference for all Types example**
 
-This example is a class named `FullUsesTypes` wichs its methods should use 
+This example is a class named `Java10var` whose methods should use 
 `Type Inference` to declare variables using them in Expressions, Statements, and Blocks as Loops.
 
 Perform the below commands to see the contents of the respective Java classes contained in the src folder:
 
-    $ cat src/com/typeinference/FullUsesTypes.java
+    $ cat src/com/typeinference/Java10var.java
```

Comment:
Have you had a look at what I have done with Java10var class? You might need to update your class file.

Reply:
Hi @neomatrix369 what do you mean?  Is it about the switch? I will sent a new PR

Reply:
Ignore me I realised you have just called the file `Java10var`, it's not the same as the one I called.